### ISSUE TITLE
Channel not automatically adding

### ISSUE THREAD

**Issue by Tolkien-Pendergast** [Chalsean/XFaction#362]
Hiya! We've recently adopted your great add-on to assist with our WoW Community (Valhalla, spanning across Khaz/Dath/Aman/Barth/Frost and Alliance sides.

Overall it has been a great success, however there have been intermittent reports of the channel not auto-adding. I have had the users disable all addons, re-install the mod and try again.

It shows the correct guild information in the setup, but still nothing in the channels. Is this something you might have an idea on? The common denominator could be the 5 channel (ValDra) and trhe other users already have a 5 channel too.
![Expected Channel](https://user-images.githubusercontent.com/132997535/237053040-8767d286-1d49-4173-88d5-105edfdb45b6.png)
![Not working channel](https://user-images.githubusercontent.com/132997535/237053047-66d814c7-2370-4d92-bd19-e750ee6aea17.png)
![Properly setup Guild](https://user-images.githubusercontent.com/132997535/237053051-350376f8-d42d-441a-8e5f-cc7caf1aa588.png)

Thanks!

Please see the two screenshots :)

**Comment by Cameroncodes92** [comment]
I am also an officer in the above guild. I Just want to add something further with the 2 members that have had this issue. 

They had Elvui enabled when they first installed Xfaction, I am not sure if that has prevented the channel creation and even when disabling all addons the channel does not create at all. 
Is there another way or command to brute force the channel creation?

I hope you all can help or give us a direction to check.

**Comment by Tolkien-Pendergast** [comment]
Just another point to add to Cameron, all the ELVUI profiles were out of date upon installing the addon. Could this have caused a backround disruption?

**Comment by Tolkien-Pendergast** [comment]
Hi, so it looks like Aman'thul is not linked to Khaz'Goroth and Dath'Remar on the addon, though they are linked in blizzard realms. Is this something that could be looked at? :)

**Comment by Cameroncodes92** [comment]
Hi there again, we have located the bug. We tried to push the code changes for your review but its locked, no worries :) See screnshot below: 

TLDR: Aman'Thul is part of the OCE connected servers Khaz/Dath. 

![image](https://github.com/Chalsean/XFaction/assets/54254173/3809c2cc-1473-4db6-adf7-2fa1e68fce6a)

Hopefully we can get this pushed and updated asap. Thank you.

**Comment by Chalsean** [comment]
Thanks for finding the issue!  I'll push a beta build (4.5.7) with the updated info.  Once you validate it works for you, I can push it to production.

**Comment by Cameroncodes92** [comment]
Hi Chalsean, thank you for pushing this Beta. I just tested and can confirm it works as expected. I created a toon on Aman and also had others download Beta version. Thank you so much for the quick turn around here. Screenshot is from Aman toon showing connections: 

![image](https://github.com/Chalsean/XFaction/assets/54254173/c4363815-86c6-45b9-9a93-eeeb48b45944)

**Comment by Chalsean** [comment]
Fixed in 4.5.8
